library/src/main/java/com/google/android/material/motion/runtime/TargetScope.java

```diff
@@ -118,24 +132,42 @@ private int getDetailedState() {
     return state;
   }
 
+  private void removeNamedPlan(NamedPlan plan, String name, Performer.NamedPlanPerformance performer) {
+    if (performer != null) {
+      performer.removePlan(plan, name);
```

Comment:
remove `plan` param from `NamedPlanPerformance#removePlan()` and this `removeNamedPlan()` method, according to spec https://material-motion.gitbooks.io/material-motion-starmap/content/specifications/runtime/named-plans.html

The expectation is that the Performer will keep track of its own named Plans, if necessary (we may change this in the future, but remove for now).